frontend/src/components/network-import-form/network-import-form.tsx

```diff
@@ -86,81 +75,49 @@ export function NetworkImportForm({ onComplete }: NetworkImportFormProps) {
   }
 
   return (
-    <form onSubmit={handleSubmit(submit)}>
+    <form
+      onSubmit={handleSubmit(values => submit({ ...values, network: 'other' }))}
+    >
       <FormGroup
-        label='Network'
-        labelFor='network'
-        intent={errors.network?.message ? Intent.DANGER : Intent.NONE}
-        helperText={errors.network?.message}
+        label='URL or path'
```

Comment:
suggestion(non-blocking): shouldn't the props be double quoted to mimic more closely html? I thought this was set via eslint. https://stackoverflow.com/questions/44208081/js-quote-best-practices-es6-react-single-double-backticks

Reply:
created an issue for it, as imo is not related to this PR: https://github.com/vegaprotocol/vegawallet-desktop/issues/352